usr/bin/revpi-btuart

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+# referenced pi-bluetooth, bluetooth attach for Revolution Pi BT
+
+HCIATTACH=/usr/bin/hciattach
+TTYDEVNAME="$1"
+
+$HCIATTACH $TTYDEVNAME bcm43xx 3000000 flow
```

Comment:
Please use `""` around `$TTYDEVNAME`

Reply:
The Flat devicetree limits the speed to 2000000. If you are sure that this is not correct please create a ticket for the flat device tree that this is changed. Otherwise this should be limited here to 2000000 as well. But this should depend on the device type flat or atlease add a comment to extend this, if future/other devices with BT will be supported.